doc/pages/options.asciidoc

```diff
@@ -79,6 +79,11 @@ are exclusively available to built-in options.
     as a string but the set commands will complain if the entered text
     is not a valid regex
 
+*key*::
+    a single keypress using the same syntax as `map` (see
+    <<mapping#mappable-keys,`:doc mapping mappable-keys`>>).  If
+    multiple keys are entered, only the first will be used.
```

Comment:
> If multiple keys are entered, only the first will be used.

I think it's better to throw  a `runtime_error{"too many keys"}`, then we can delete this line from docs.

Reply:
I changed it to require exactly 1, following the example of codepoint.  However, I just realized that does prevent determined users from disabling it altogether if they want.  Should `set global interrupt_key ''` be allowed, with a caveat of "be it upon your own head"?